cypress/integration/TSP_Tests/e2e_spec.js

```diff
@@ -0,0 +1,26 @@
+// Context for E2E testing
```

Comment:
awesome job on this love the tests you've added here

Reply:
😎